### PR TITLE
Helm quickstart

### PR DESCRIPTION
**Pull request by mproch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +24,33 @@ After doing it, you can will have available:
 * [Grafana](http://localhost:8081/grafana/)
 * [AKHQ](http://localhost:8081/akhq/)
 
+## Running kubernetes + helm mode mode
```
[the remark this answers is not in the document]
Comment (mproch) [reply README.md]: :+1:

File: README.md [hunk 2]
```diff
@@ -3,15 +3,19 @@
...
 This is a part of Nussknacker's [quickstart](https://nussknacker.io/quickstart/docker).
-On this page you can find information about how to start work with Nussknacker designer tool.
-If you want to start your own project using Nussknacker, just fork this repo, change `./conf/nusskacker/nussknacker.conf`,
-register own schemas (similar to those available at `./testData/schema`) and start designing own scenarios!
+On this page you can find information about how to start work with Nussknacker Designer tool.
+If you want to start your own project using Nussknacker, just fork this repo, adjust configuration,
+register own schemas (similar to those available at `./common/schemas`) and start designing own scenarios!
 
-## Running
+Available modes:
```
[the remark this answers is not in the document]
Comment (mproch) [reply README.md]: showcases sound ok, changed

File: common/scripts/displayErrors.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (mproch) [reply common/scripts/displayErrors.sh]: changed to displayLogs.sh

File: README.md [hunk 4]
```diff
@@ -20,6 +24,33 @@ After doing it, you can will have available:
...
+
+We assume that you have configured `kubectl` and `helm`. 
+
+### Local K3d installation
+If you don't have K8s available, we recommend installing [k3d](https://k3d.io/).
+Instructions below assume that cluster was created with ingress port mapped to 8081 - see [guide](https://k3d.io/v5.0.0/usage/exposing_services/#1-via-ingress-recommended) for details.
+
+### Installation
+
+First, set `RELEASE` environment variable, e.g to `nu-quickstart`, which will be used as Helm release name. It's also needed for test data scripts.
```
[the remark this answers is not in the document]
Comment (mproch) [reply README.md:37]: no, it's not standard. The main reason for setting this explicitly is that I use existence of this env var to distinguish in scripts - if it's k8s-helm or docker. Do you have any better way to do it, without having to copy all scripts?

[the remark this answers is not in the document]
Comment (mproch) [reply README.md:37]: :+1: I did sth like this

[the remark this answers is not in the document]
Comment (arkadius) [reply README.md]: nice, thx

**Review by arkadius — APPROVED** [review]

**Pull request merged**